DevelopedCalculator.js

```diff
@@ -1,5 +1,5 @@
 class DevelopedCalculator extends Calculator {
-    history = null;
+    history = 0;
```

Comment:
this is array 
`history = [ ] `

Reply:
it needs to be array to hold more than on value 

everytime, a button is clicked, we need to push new value to this array 

the new value is `this.currentResult`

Reply:
does this make sense?

Reply:
Yes